src/platform/windows/display_vram.cpp

```diff
@@ -434,6 +434,7 @@ class hwdevice_t : public platf::hwdevice_t {
 
     this->device_ctx_p = device_ctx_p;
 
+    DXGI_FORMAT src_format = format;
```

Comment:
@cgutman

Related to #609: I actually made an additional mistake here: I assumed that `format` was initialized with the correct value passed from `display_base_t::init`, but in fact it is just an uninitialized variable that resolves to `DXGI_FORMAT_UNKNOWN`, which coincidentally seems to handle both `DXGI_FORMAT_R8G8B8A8_UNORM` and `DXGI_FORMAT_B8G8R8A8_UNORM`. This is obviously not ideal and is supposed to select the proper format based on a similar `img->is_bgr` check as done here: https://github.com/LizardByte/Sunshine/blob/62f98b841cc5e44b261ae63a39e1bdacb9d70b9a/src/platform/windows/display_ram.cpp#L288

However, looking at #609, instead of fixing this, it looks like I should just remove both the `vram` and `ram` commits entirely in place of #609? I will not be able to edit the PR and verify in depth for a few hours, but I will be happy to back out those commits ASAP if you can verify my suspicion.

Reply:
I think `DXGI_FORMAT_UNKNOWN` tells D3D11 to use the actual format of the image (which is what we want).

The base `IDXGIOutput5` part needs more work to handle the proper surface format selection. I think we want to avoid the HDR formats if we're not in HDR mode. I'll take a stab at it after the holidays.